File: FindMaximum.py
# Find Maximum
# - Given a list of integers, find the maximum value.
# - You can use the built-in `max()` function or implement it manually with a loop.

nums = [3, 7, 2, 10, 4]
maximum = nums[0]

for num in nums[1:]:
    if num > maximum:
        maximum = num

File: FindMaximum.cs
// Find Maximum
// - Given an array of integers, find the maximum value.
// - You can use the built-in methods or implement it manually with a loop.

using System;

class Program
{
    static void Main(string[] args)
    {
        int[] nums = {3, 7, 2, 10, 4};  
        int maximum = nums[0];       

        for (int i = 1; i < nums.Length; i++)  
        {
            if (nums[i] > maximum)          
            {
                maximum = nums[i];           
            }
        }

        Console.WriteLine($"Maximum value is: {maximum}");
    }
}
